main.go

```diff
@@ -312,6 +340,15 @@ func main() {
 		Host:   hostport,
 		Path:   "/",
 	})
+
+	httpsd := &http.Server{
+		Handler:        r,
+		Addr:           httpAddr,
+		WriteTimeout:   httpTimeout,
+		ReadTimeout:    httpTimeout,
+		MaxHeaderBytes: maxHeaderBytes,
+	}
+
```

Comment:
I need to check out the PR on my own machine but this looks like it's in the wrong place.